### PR TITLE
Heatmap chart

### PR DESCRIPTION
**Pull request by EmreCanHaciosmanoglu**

### CODE REVIEW

File: src/components/heatmap-chart/HeatmapChart.tsx [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+                fill={
+                    'rgb(' +
+                    (255 - Map(i[2], Small(s), Big(s), 0, 255)) +
+                    ',' +
+                    (255 - Map(i[2], Small(s), Big(s), 0, 255)) +
+                    ',255)'
+                }
+            />
+            {(s.dataLabels.enabled) ? (
+                <text
```
Comment (Samoxive) [inline src/components/heatmap-chart/HeatmapChart.tsx]: We can use the common chart-label css class for <text> elements for consistency.

Comment (EmreCanHaciosmanoglu) [reply src/components/heatmap-chart/HeatmapChart.tsx]: I didn't get it

Comment (EmreCanHaciosmanoglu) [reply src/components/heatmap-chart/HeatmapChart.tsx]: you mean--- className="chart-label" ---

Comment (Samoxive) [reply src/components/heatmap-chart/HeatmapChart.tsx]: Yup!

**Pull request merged**